README.md

```diff
@@ -817,6 +817,26 @@ when you need to retrieve a single record by some attributes.
   Mitigate code duplication by using partial templates and layouts.
 <sup>[[link](#partials)]</sup>
 
+* <a name="erb-indentation"></a>
+  Use the same indentation style in ERB as you would in [plain Ruby](https://github.com/bbatsov/ruby-style-guide#no-double-indent). <sup>[[link](#erb-indentation)]</sup>
+
+  ```erb
+  <%# starting point (line is too long) %>
+  <%= form_for @post, as: :post, url: posts_path, html: { class: "new_post", id: "new_post" } do |f| %>
+
+  <%# bad (continued lines are outdented) %>
+  <%= form_for @post,
+    as: :post,
+    url: posts_path,
+    html: { class: "new_post", id: "new_post" } do |f| %>
+
+  <%# good (normal indent) %>
```

Comment:
Sure thing, I've added an example with aligned params similar to the referenced item in the Ruby Style Guide.

Thanks for the feedback!